src/controllers/Controller.py

```diff
@@ -0,0 +1,4 @@
+class Controller:
```

Comment:
Just wondering here: is this meant to be an abstract class?

Reply:
Lol, yesh. It came from an unfinished bigger project, where there would also be ETFControllers and other financial products. But as you might be able to see it was not in the stadium where those this were important yet. If you do have experience with abstract classes in python I would love some pointers :)